DataConnectors/Connector_syslog_WatchGuardFirebox.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "WatchguardFirebox",
+    "title": "Firebox",
+    "publisher": "Watchguard",
+    "descriptionMarkdown": "Syslog is an event logging protocol that is common to Linux. Applications will send messages that may be stored on the local machine or delivered to a Syslog collector. When the Agent for Linux is installed, it configures the local Syslog daemon to forward messages to the agent. The agent then sends the message to the workspace.\n\n[Learn more >](https://aka.ms/sysLogInfo)",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **<enter the Kusto Function alias>** in queries and workbooks. [Follow steps to get this Kusto Function>](<Link to Kusto Function on Azure Sentinel GitHub>)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Watchguard",
+            "baseQuery": "Firebox"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Firebox log category count in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by SeverityLevel\n"
+        },
+        {
+            "description" : "Firebox top 10 messages in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by SyslogMessage\n| top 10 by count_ desc"
+        },		
+		{
+            "description" : "Firebox top 10 HostIP in last 24 hours",
+            "query": "Firebox\n | where HostName contains 'WatchGuard'\n| where TimeGenerated >= ago(24h)\n| summarize count() by HostIP\n| top 10 by count_ desc"
+        }	
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(Firebox)",
+            "lastDataReceivedQuery": "Firebox\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "Firebox\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {

```

Comment:
Set preview flag for now:

"availability": { 
            "status": 1,
            "isPreview": true
            },